gem/views.py

```diff
@@ -60,3 +62,53 @@ def form_valid(self, form):
         authed_user = authenticate(username=username, password=password)
         login(self.request, authed_user)
         return HttpResponseRedirect(form.cleaned_data.get('next', '/'))
+
+
+class GemRssFeed(Feed):
+    title = 'GEM Feed'
+    description = 'GEM Feed'
+    description_template = 'feed_description.html'
+
+    def __call__(self, request, *args, **kwargs):
+        self.base_url = '{0}://{1}'.format(request.scheme, request.get_host())
+        return super(GemRssFeed, self).__call__(request, *args, **kwargs)
+
+    def get_feed(self, obj, request):
+        feed = super(GemRssFeed, self).get_feed(obj, request)
+        # set language to Tagalog instead of settings.LANGUAGE_CODE
+        feed.feed['language'] = 'tl'
```

Comment:
Hmm..this is not going to work - we have 5 sites that are based on this repo.
I think you can use 
```python
language = SiteLanguage.objects.filter(is_main_language=True).first()
if language:
    feed.feed['language'] = language.locale
else:
    feed.feed['language'] = 'en'
```